.github/workflows/Component.BuildTest.yml

```diff
@@ -20,7 +20,7 @@ on:
         required: false
         type: string
       os-list:
-        default: '[ "windows-latest", "ubuntu-22.04", "ubuntu-22.04-arm" ]'
+        default: '[ "windows-latest", "windows-11-arm", "ubuntu-22.04", "ubuntu-22.04-arm" ]'
```

Comment:
As everything else now has a specific version, maybe we should swap to `windows-2025` too? Since a few days ago they'll be rolling over from 2022 to 2025 with `latest` anyway.

Reply:
Separate PR?
As I remember I have pinned ubuntu-22.04 (x64) due to issues on the newer version.
Arm are not available as latest.